projects/igniteui-angular/src/lib/directives/scroll-inertia/scroll_inertia.directive.spec.ts

```diff
@@ -359,9 +371,8 @@ export class ScrollInertiaComponent implements OnInit {
    public ngOnInit() {
         this.scrInertiaDir.IgxScrollInertiaScrollContainer = this.scrollContainer.nativeElement;
 
-        this.scrollContainer.nativeElement.addEventListener('scroll', (evt) => {
-            this.onScroll(evt);
-        });
+        this.onScroll = this.onScroll.bind(this);
```

Comment:
Bind the onScroll in the template instead IMO

Reply:
You're right. Done.